protocol/app/app.go

```diff
@@ -603,7 +602,7 @@ func New(
 
 		// Non-validating full-nodes have no need to run the price daemon.
 		if !appFlags.NonValidatingFullNode && daemonFlags.Price.Enabled {
-			exchangeStartupConfig := configs.ReadExchangeStartupConfigFile(homePath)
+			exchangeStartupConfig := constants.StaticExchangeQueryConfig
```

Comment:
For my understanding:
- This PR is no-op for the actual value of `exchangeStartupConfig` right? Meaning it's still initialized as the same value as before, just not read from a config file
- Is there currently any sanity check for the `exchangeStartupConfig` being populated correctly?

Reply:
Yes and yes. See [this file](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/daemons/pricefeed/client/constants/static_exchange_startup_config_test.go).